examples/design-system/apps/docs/.storybook/main.js

```diff
@@ -0,0 +1,27 @@
+const path = require("path");
+
+module.exports = {
+  stories: ["../stories/**/*.stories.mdx", "../stories/**/*.stories.tsx"],
+  addons: ["@storybook/addon-links", "@storybook/addon-essentials"],
+  framework: "@storybook/react",
+  core: {
+    builder: "@storybook/builder-vite",
+  },
+  async viteFinal(config, { configType }) {
+    // customize the Vite config here
+    return {
+      ...config,
+      resolve: {
+        alias: [
+          {
+            find: "@acme-core",
```

Comment:
Can we make this `@acme/core` instead of `@acme-core`? Or is there a reason for the hyphen